app-backup/tob/tob-0.26-r2.ebuild

```diff
@@ -7,8 +7,8 @@ EAPI=5
 inherit eutils
 
 DESCRIPTION="A general driver for making and maintaining backups"
-HOMEPAGE="http://tinyplanet.ca/projects/tob/"
-SRC_URI="http://tinyplanet.ca/projects/tob/${P}.tgz"
+HOMEPAGE="https://web.archive.org/web/20110109221843/http://tinyplanet.ca/projects/tob/"
```

Comment:
I am not sure if using the wayback machine like this makes sense. If the homepage is unavailable, per [devmanual](https://devmanual.gentoo.org/ebuild-maintenance/index.html#homepage-unavailable) we should just point to https://wiki.gentoo.org/wiki/No_homepage